src/main/resources/de/qabel/desktop/ui/remotefs/remotefs.fxml

```diff
@@ -18,6 +18,11 @@
 
     <children>
       <HBox prefHeight="100.0" prefWidth="200.0">
+          <Button mnemonicParsing="false" onAction="#handleDownloadButtonAction" text="Download">
```

Comment:
hint: if you add css `-fs-spacing: 10px` all buttons in the HBox get their insets automatically